### DISCUSSION TITLE
Long note % (Auto host)

### DISCUSSION THREAD

**Discussion by its54ig** [Quaver/Quaver#4771] [Feature Request]
### Is your feature request related to a problem? Please describe.

No problems really.... 

### Describe the solution you'd like

Make the autohost also pick maps within the Long Note % of the multiplayer lobby.

**Comment by its54ig** [comment]
And also max song length

**Comment by Nexility0** [comment]
this is already implemented check the wiki here: https://wiki.quavergame.com/docs/multiplayer/commands

**Comment by its54ig** [comment]
@Nexility0 , I know you can use all of those commands to customize your room, but the QuaverBot will only pay attention to difficulty range and the mode. demonstrated in this screenshot:

https://i.imgur.com/P2CQHSz.png

**Comment by its54ig** [comment]
and if you're wondering, I also even tested it out on a private lobby.

**Comment by ichbin54p** [comment]
Hi, this is me on my new account. My old account was lost due to 2fa.

**Comment by its54ig** [comment]
I got my account back
